src/main/webapp/resources/js/pages/password/ForgotPassword.jsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Button, Form, Input } from "antd";
+import { LoadingOutlined, MailOutlined } from "@ant-design/icons";
+import { InfoAlert } from "../../components/alerts";
+import { SPACE_MD } from "../../styles/spacing";
+import { blue6 } from "../../styles/colors";
+import { useCreatePasswordResetEmailMutation } from "../../apis/password-reset";
+
+const { Item } = Form;
+
+/**
+ * React component to render the forgot password form
+ * @param {function} updateDisplayLoginPage Function to update whether to display login page
+ * @returns {*}
+ * @constructor
+ */
+export function ForgotPassword({ updateDisplayLoginPage }) {
+  const [forgotPassword] = useCreatePasswordResetEmailMutation();
+  const [forgotPasswordForm] = Form.useForm();
+  const [loading, setLoading] = React.useState(false);
+  const [message, setMessage] = React.useState(null);
+  const usernameOrEmailRef = React.useRef();
+
+  /**
+   * When the component gets added to the page,
+   * focus on the usernameOrEmail input.
+   */
+  React.useEffect(() => {
+    usernameOrEmailRef.current.focus();
+    usernameOrEmailRef.current.select();
+  }, []);
+
+  const submitForgotPasswordForm = () => {
+    setLoading(true);
+    setMessage(null);
+
+    forgotPassword({
+      usernameOrEmail: forgotPasswordForm.getFieldValue("usernameOrEmail"),
+    }).then((response) => {
+      if (response.error) {
```

Comment:
Why are you not handling this in a catch?

Reply:
For whatever reason it's not catching an error because the error is in the response. If I remove the response.error code and add the catch it just has an error in the console that it can't read `data` in `setMessage(response.data.message);`

Reply:
It's not working because you are trying to use it as a regular ajax request.  I just looked through the code and realized that this is actually a `redux-toolkit` mutation.  In your declaration on line 18 you could do the following:

```javascript
const [forgotPassword, {error}] = useCreatePasswordResetEmailMutation();
```

or to keep it the way you have (I am surprised what you are doing is working because you are not unwrapping the response):

```javascript
forgotPassword({
      usernameOrEmail: forgotPasswordForm.getFieldValue("usernameOrEmail"),
  })
  .unwrap() // <------- MISSING THIS
  .then((response) => {
     setMessage(response.data.message);
      forgotPasswordForm.resetFields();
      setLoading(false);
    })
  .catch(error -> setMessage(error);
```

Reply:
Unwrapping it worked! Thanks! Updated in 8e07449499d3a3b0135b0350f8ebe2f73c2dfc76